packages/@aws-cdk/aws-ec2/lib/client-vpn-authorization-rule.ts

```diff
@@ -36,7 +36,14 @@ export interface ClientVpnAuthorizationRuleProps extends ClientVpnAuthorizationR
   /**
    * The client VPN endpoint to which to add the rule.
    */
-  readonly clientVpnEndoint: IClientVpnEndpoint;
+  readonly clientVpnEndpoint: IClientVpnEndpoint;
```

Comment:
Aha, here is the confusion. Both need to be declared optional (type-wise), and runtime code needs to validate exactly one of them is supplied anyway.

Don't forget that both need a `@default` annotation as well. I'd recommend something like:

```
@default clientVpnEndpoint is required
```